voorhoede/head-start
# Upgrade astro and cloudflare adapter to latest version

# Changes

- Upgrade astro, astrojs/cloudflare and astrojs/check
- Changes related to [https://github.com/withastro/adapters/blob/main/packages/cloudflare/CHANGELOG.md#1000](https://github.com/withastro/adapters/blob/main/packages/cloudflare/CHANGELOG.md#1000) and [https://github.com/withastro/adapters/blob/main/packages/cloudflare/CHANGELOG.md#processenv](https://github.com/withastro/adapters/blob/main/packages/cloudflare/CHANGELOG.md#processenv)
- Fixed some linting configuration and linting errors/warnings

# Associated issue

Resolves #150 

# How to test

1. Open [https://preview-throws-500-due-to-as.head-start.pages.dev/en/](https://preview-throws-500-due-to-as.head-start.pages.dev/en/)
2. The homepage should be previewable by entering the secret, where [https://preview.head-start.pages.dev/en/](https://preview.head-start.pages.dev/en/) failed with a 500
